include_physics/ideal_MHD.h

```diff
@@ -387,14 +387,300 @@ inline void init_fields(uint ix, uint iy, uint iz, global REAL* u) {
 // Surface terms
 //--------------------------------------------------------------------------------------------------
 
+#ifndef USE_PERIODIC
+
+#define sq(x) ((x)*(x))
+#define PLUS(x) (max(x, 0.0))
```

Comment:
Shouldn't this be `fmax` in order to allow floating point numbers? Or is it just used for integers?

Reply:
This macro is only used with floats. I changed it and will commit the change later.